modules/rostests/winetests/msi/CMakeLists.txt

```diff
@@ -23,6 +23,10 @@ list(APPEND SOURCE
     suminfo.c
     precomp.h)
 
+if(VS_SOLUTION)
```

Comment:
If you end up needing this check... MSVC_IDE should do the trick. I don't think you need a new variable.

Reply:
Right, I assumed cmake `MSVC_IDE` meant its visual interface, whereas it is simply similar to configure.cmd `VS_SOLUTION` build tool.
Done.